todo_app/test/test_unit.py

```diff
@@ -0,0 +1,21 @@
+"""Unit tests for bank.py"""
+
+from todo_app.item import Item
+import pytest
+from todo_app.view_model import ViewModel
+
+
+def test_get_ToDo_Trello():
+    viewModel = ViewModel(items=[Item( "60afa040b07e9a4371098530","title", "ToDo")])
+    list= viewModel.todo_items
+    assert len(list)==1
```

Comment:
It would be good to expand these tests, these would pass even if your ViewModel methods all just returned the whole list!